daffodil-core/src/main/scala/org/apache/daffodil/dsom/SchemaSetIncludesAndImportsMixins.scala

```diff
@@ -57,13 +57,20 @@ trait SchemaSetIncludesAndImportsMixin { self: SchemaSet =>
   lazy val allSchemaFiles = {
     val fd = fakeXMLSchemaDocument //bootstrap
     val sa = fd.seenAfter
+    val (_, first) = if (sa.value.isEmpty) (null, null) else sa.value.head
```

Comment:
Is it actually possible for `sa.value.isEmpty` to be true? Seems like we must always have at least one value since we must be compiling something. If so, could this be simplified to this:
```scala
val first = sa.value.head._2
```
Then we don't this `null` stuff, which we tend to avoid in scala.

Reply:
Updated with the recommended change.